cargo/cargo_bootstrap.bzl

```diff
@@ -55,12 +55,17 @@ def cargo_bootstrap(
     if build_mode == "release":
         args.append("--release")
 
+    env = {}
+
+    if repository_ctx.attr.include_perl_on_windows and "win" in repository_ctx.os.name:
```

Comment:
Instead of always using setting `PERL` here, could the `cargo_bootstrap_repository` rule take an `env` section which allows users to set these?

Reply:
I added a `windows_env_vars_to_labels` which does this _just on Windows_, but it feels still pretty similarly specific...

Reply:
`windows_env_vars_to_labels` to me is an improvement but not quite what I had in mind. Would something like https://github.com/bazelbuild/rules_rust/pull/932 work for you?

Reply:
Works for me! Though exactly which triple I should be using for "Whatever CI is on Windows" I'm not sure :D

Reply:
I'd probably just use:
```
x86_64-pc-windows-gnu
x86_64-pc-windows-msvc
```

Perhaps that section could be expanded to support matching on components of a triple, `system`, `vendor`, etc... Host detection is not the easiest thing to do sadly... I wish there was more in Bazel to make this easy.